source/lib/JMSLab/autofill.py

```diff
@@ -10,15 +10,15 @@ def Autofill(var, format = "{}", namespace = None):
             parent = parent.f_back
         
         if var not in parent.f_locals.keys() or parent is None:
-            raise Warning(f"Autofill: Variable '{var}' not found")
+            raise Exception(f"Autofill: Variable '{var}' not found")
         
         namespace = parent.f_locals
     
     commandname, content = var, namespace[var]
     
     return newcommand.format(commandname, content)
 
-def GenerateAutofillMacros(autofill_lists,  autofill_formats, autofill_outfile):
+def GenerateAutofillMacros(autofill_lists,  autofill_outfile, autofill_formats = "{:.2f}"):
```

Comment:
The order of the arguments looks weird here, but not sure how to fix it!

Would there be a natural default for `autofill_outfile` so that we can put it in the end? Why not `autofill_outfile = '.'`?

Reply:
Thanks @santiagohermo! Took a pass at it in [af4bbde](https://github.com/JMSLab/Template/pull/57/commits/af4bbde924d7676b5b3553f1dbc38dfbcc387c27) let me know if it makes sense.

Reply:
Look good, thanks @jinfu93!